data/styles/clear/include/Icons.mapcss

```diff
@@ -708,11 +720,13 @@ node|z12-[aeroway=aerodrome]::int_name,
 
 /*check offset int_name*/
 
-node|z11[railway=halt]
+area|z11[building=train_station],
 {icon-image: train-s.svg;icon-min-distance: 10;text-optional: true;}
-node|z12[railway=halt]
+area|z12[building=train_station],
+node|z12[railway=station],
```

Comment:
To avoid conflicts with subways. Draw icon but not for subways.
node|z12[railway=station][!transport],